docs/reference/changelog-r2020.md

```diff
@@ -4,6 +4,7 @@
 Released on XXX YYth, 2020.
 
   - Bug fixes
+    - Fixed field changes not applied in case of nested [PROTO](proto.md) nodes ([ #2063](https://github.com/cyberbotics/webots/pull/2063)).
```

Comment:
```suggestion
    - Fixed field changes not applied in case of nested [PROTO](proto.md) nodes ([#2063](https://github.com/cyberbotics/webots/pull/2063)).
```